libs/utils/src/lib/utils.ts

```diff
@@ -302,3 +302,20 @@ export function toLanguageVersionString(languages: LanguageRecord) {
     .filter((d) => d)
     .join(', ');
 }
+
+
+export function removeNulls(obj: any) {
+  const isArray = Array.isArray(obj);
+  for (const k of Object.keys(obj)) {
+    if (obj[k] === null || obj[k] === undefined) {
+      if (isArray) {
+        obj.splice(+k, 1)
+      } else {
+        delete obj[k];
+      }
+    } else if (typeof obj[k] === "object") {
+      removeNulls(obj[k]);
+    }
+  }
+  return obj;
+}
```

Comment:
Better to treat the array in another way than the object
```suggestion
export function removeNulls(obj: any) {
  if (Array.isArray(obj)) return obj.map(removeNulls).filter(el => el !== null && el !== undefined);
  for (const k of Object.keys(obj)) {
    if (obj[k] === null && obj[k] === undefined) {
      removeNulls(obj[k]);
    } else {
      delete obj[k];
    }
  }
  return obj;
}
```